### PR TITLE
enforcer options: add namespace enforcement mode new option

### PR DESCRIPTION
**Pull request by hectorj2f**
## Type of change

Add new documentation

### What should this PR do?

Add documentation on how to use the enforcer option: namespace_enforcement_mode and its values.

related to this https://github.com/chainguard-dev/customer-issues/issues/105

### CODE REVIEW

**Comment by hectorj2f** [comment]
Note I added a Do Not Merge until we merge the PR with the functionality in mono.

**Comment by hectorj2f** [comment]
This is ready for a review.

**Review by SharpRake — APPROVED** [review]
LGTM! Thanks so much Hector

**Pull request merged**
